docs/source/reference/volumes.rst

```diff
@@ -335,6 +338,103 @@ When you terminate the cluster, the ephemeral volumes are automatically deleted:
   $ sky down my-cluster
   # Cluster and its ephemeral volumes are deleted
 
+Mount PVCs
+~~~~~~~~~~
+
+In addition to creating and mounting volumes via SkyPilot's volume management, you can also mount existing PersistentVolumeClaims (PVCs) directly to SkyPilot clusters and managed jobs.
```

Comment:
We should always say why people may want to do this compared to the previous approach, instead of just listing out the option here...

```suggestion
Using SkyPilot volumes allows you to mount different volumes to different tasks. SkyPilot also offers an advanced way to mount a Kubernetes PVC with the detailed Kubernetes configs. This allows you to:
1. Mount a PVC with additional configurations that is not supported by SkyPilot volumes
2. Specifying a global (per Kubernetes context) PVC to be mounted on all SkyPilot clusters.
```

For the following two examples, we can name them as: a) `Mount a PVC with additional configuration`; b) `Mount a PVC to all clusters in each context`